src/modules/infobox/components/styling/DrawingStyleIconSelector.vue

```diff
@@ -131,6 +147,15 @@ export default {
         changeDisplayedIconSet(dropdownItem) {
             this.currentIconSet = dropdownItem.value
         },
+        getImageStrokeStyle(isColorable, isSelected, color) {
+            if (isColorable) {
+                return {
+                    filter: `drop-shadow(1px 1px 0 ${color.border}) drop-shadow(-1px -1px 0 ${color.border})`,
+                }
+            } else if (isSelected) {
+                return { filter: 'drop-shadow(0px 0px 0 white)' }
+            }
+        },
```

Comment:
I've tested on iOs, and also Safari desktop, and Safari is really playing with us here... The rule was CSS clearly set when I've played with colors, so on the CSS part everything was correct but the rendering was not. I then unchecked, and checked back the rule in the CSS inspector, and voilà, it was now correctly rendered.
So yeah, Safari issue, we can't do much about it...

Reply:
I imagine you've tested both ways, having the filter defined in the SCSS part, or here?
If both are not doing a good job on Safari, and thus we can just ignore this ***** browser, I would prefer that we define a class here, and not directly CSS rules.

Reply:
I leave it like this as discussed, this allow to not touch the component if the `border.color` changes in the future.